pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/kmp/factories.ios.kt

```diff
@@ -205,15 +205,15 @@ private fun mapAppContextEvent(from: PubNubAppContextEventObjC?): PNObjectEventM
                 type = from.type(),
                 data = PNUUIDMetadata(
                     id = from.metadata().id(),
-                    name = PatchValue.of(from.metadata().name()),
-                    externalId = PatchValue.of(from.metadata().externalId()),
-                    profileUrl = PatchValue.of(from.metadata().profileUrl()),
-                    email = PatchValue.of(from.metadata().email()),
-                    custom = PatchValue.of(from.metadata().custom()?.safeCast()),
-                    updated = PatchValue.of(from.metadata().updated().orEmpty()),
-                    eTag = PatchValue.of(from.metadata().eTag().orEmpty()),
-                    type = PatchValue.of(from.metadata().type()),
-                    status = PatchValue.of(from.metadata().status())
+                    name = if (from.metadata().hasName()) { PatchValue.of(from.metadata().name()) } else { null },
```

Comment:
You could consider changing here in all other places to sth like:
from.metadata.name()?.let { nonNullMetadata -> PatchValue.of(nonNullMetadata) }
or
from.metadata.name()?.let { PatchValue.of(it) }

Reply:
Scratch this. My sugesstion is not valid in this case.